web/modules/custom/joinup_federation/tests/modules/joinup_federation_test/src/Plugin/pipeline/Pipeline/JoinupFederationTestingPipeline.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Drupal\joinup_federation_test\Plugin\pipeline\Pipeline;
+
+use Drupal\joinup_federation\JoinupFederationPipelinePluginBase;
+
+/**
+ * Provides a pipline testing plugin.
+ *
+ * @PipelinePipeline(
+ *   id = "joinup_federation_testing_pipeline",
+ *   label = @Translation("Joinup federation testing pipeline"),
+ *   steps = {
```

Comment:
True. As long as we don't perform pipeline operations, like `::next()`, `::current()`, etc. this will work. Otherwise an empty step list will throw an exception in `\Drupal\pipeline\Plugin\PipelinePipelinePluginBase::getStepList()`.